ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/sync/PipelineChainDownloaderTest.java

```diff
@@ -119,6 +117,7 @@ public void shouldUpdateSyncStateWhenTargetSelected() {
 
   @Test
   public void shouldRetryWhenSyncTargetSelectionFailsAndSyncTargetManagerShouldContinue() {
+    immediatelyCompletePauseAfterError();
```

Comment:
YES! Wonder if this would fix the FullSyncChainDownloaderTest that always hangs locally

Reply:
It's when I run it as part of ./gradlew build
Might be an m1 issue